report/22_Process-Team-Organization.md

```diff
@@ -1 +1,5 @@
-How is the team organized?
+The team has organized itself mainly through a weekly meeting after the lecture in which tasks were distributed. The team has not found any formalized process corresponding to this, but it has had several advantages and drawbacks.
```

Comment:
This starts in the same way as 2.3 I would reword it here.
I would not start the next sentence with "The Team", you cant just swap we out with the team. It has the same awkwardness. 

> No formal process corresponding to our method of work could be found.

Also the method needs more explanation than a single line saying we only organized through that meeting as that simply is not true. For example we had no leader and every member in the team was equal. We worked when we had time. 
Assigning work using github issues is also a method of Team Organization as its clear who works on what. We also avoided working on each other branches and let changes needed to be done to a PR be done by the developer who made the PR.